### PR TITLE
Install & update systemd influxdb service

### LINKED ISSUE #3989: [0.9.3] Service fails to start on Ubuntu 15.04
Just did a clean install on a new VM with Ubuntu 15.04
Starting the service fails:

```
sudo service influxdb start
Failed to start influxdb.service: Unit influxdb.service failed to load: No such file or directory.
```

Same issue with telegraf 0.1.7

This is probably related to systemd having replaced Upstart as the standard boot and service manager on Ubuntu 15.04

### PR DESCRIPTION
**Pull request by sparrc**
Closes #3989

### CODE REVIEW

File: package.sh [hunk 1]
```diff
@@ -244,19 +245,32 @@ ln -s $INSTALL_ROOT_DIR/versions/$version/influxd $INSTALL_ROOT_DIR/influxd
...
+    ln -sfn $INSTALL_ROOT_DIR/init.sh /etc/init.d/influxdb
+    chmod +x /etc/init.d/influxdb
+    if which update-rc.d > /dev/null 2>&1 ; then
+        update-rc.d -f influxdb remove
+        update-rc.d influxdb defaults
+    else
+        chkconfig --add influxdb
+    fi
+
+    chown -R -L influxdb:influxdb $INSTALL_ROOT_DIR
```
Comment (otoolep) [inline package.sh]: These lines seem duped with the lines below.

**Comment by otoolep** [comment]
Package will need to be tested with this script before I can be merged. I can help if necessary.

Once the duped lines have been removed, packaging works, and the resultant deb does the right when installed on an Ubuntu 15 box, +1

**Comment by otoolep** [comment]
+1 on latest changes.

**Comment by sparrc** [comment]
I tested making a package from this branch and it installs on 15.04, service is then controllable via `systemctl [ start | stop | restart ] influxdb`

**Pull request closed without merging**